tests/flytekit/unit/core/test_wrapping.py

```diff
@@ -0,0 +1,17 @@
+from flytekit import task, workflow
+
+
+def test_task_correctly_wrapped():
```

Comment:
@bstadlbauer , we allow for stacking of @task decorators (e.g. https://github.com/flyteorg/flytekit/blob/master/tests/flytekit/unit/core/functools/stacked_decorators.py). Can you add a unit test to confirm the behavior too?

Reply:
@eapolinario I've added the corresponding test, seems to work as expected. 

Also, I wasn't sure where to place the tests, so if you feel like there would be a better spot, just let me know.

Reply:
leaving them here is fine.